<launch>

	<include file="$(find lwr_ros_control)/launch/lwr_control.launch"/>
	
	<!-- ToDo: avoid the terse_roscpp approach, load and parse parameters /-->
	<param name="tip_joint" value="lwr_arm_6_joint"/>

	<!-- hardware address /-->
	<param name="port" value="49939"/>
	<param name="ip" value="192.168.0.10"/>

	<!-- launch the real hardware interface /-->
	<node name="lwr_hw_interface" pkg="lwr_ros_control" type="lwr_server" respawn="false" output="screen"/>
    
</launch>